### DISCUSSION TITLE
F' C++ Review From Jason Turner

### DISCUSSION THREAD

**Discussion by astroesteban** [nasa/fprime#843] [General]
Hey y'all. This post is a bit random but I was wondering if any of y'all had seen Jason Turner's [video](https://youtu.be/RbhufLudVsI) reviewing the F' software from a C++ trainer's perspective?

Jason is a prominent figure in the C++ community known for his best practices tips. One of my favorite talks of his is ["Rich Code For Tiny Computers"](https://youtu.be/zBkNBP00wJE) at CppCon.

Anyways I thought his feedback was enlightening and wanted to share the video with the community in case anyone is interested.

**Comment by LeStarch** [comment]
I believe we are working through his fixes.  These updates are going  into the v3.0.0 branch given a lot of code is changing.
